packages/bsky/src/api/app/bsky/feed/getPosts.ts

```diff
@@ -31,51 +34,51 @@ export default function (server: Server, ctx: AppContext) {
   })
 }
 
-const skeleton = async (params: Params) => {
-  return { params, postUris: dedupeStrs(params.uris) }
+const skeleton = async (params: Params, ctx: Context) => {
+  const deduped = dedupeStrs(params.uris)
+  const feedItems = await ctx.feedService.postUrisToFeedItems(deduped)
```

Comment:
this does require an extra step on `getPosts` and `searchPosts` where we turn post uris into feed items first so that we can hydration information about the replyRoot along the way